clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -44,21 +45,20 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
   experience,
   fidesRegionString,
   cookie,
-  savedConsent,
   propertyId,
 }) => {
   const { i18n, currentLocale, setCurrentLocale } = useI18n();
+  const parsedCookie: FidesCookie | undefined = getFidesConsentCookie();
+  const savedConsent = window.Fides.saved_consent;
```

Comment:
@eastandwestwind I made sure the OT migration tests still pass with this change, but with the passed in `savedConsent` it was missing any GPC updates that happened between initialization and the overlay updating here. As you may recall, we run the GPC logic asynchronously _after_ `initialize()` which is where the params on this component get set.